### PR TITLE
Fix fragile test on a not case sensitive FS

### PR DESCRIPTION
**Pull request by Artpej**
TL;DR: the case unsensitive filesystems sucks :wink:.

Class resolution requires classpath walking, especially from imports resolution.
If a class is not found, then a `ClassNotFoundException` should be thrown.

for example the following test should throw a `ClassNotFoundException`:

```golo
function unknown_class = -> unknown.class
```

but , in some cases, it's a `NoClassDefFoundError` that is thrown.

This is because:
- the `gololang.Unknown` class exists
- the `Unknown.class` file is in the filesystem (not in a jar file)
- the filesystem is not case sensitive
- Jacoco use `loader.getResourceAsStream("gololang/unknown.class")` to get the bytecode to instrument
- `loader.getResourceAsStream("gololang/unknown.class")` return the content of the `"gololang/Unknown.class"` file in this case
- the JVM verify that the class that it try to load is in a file named correctly

So it seems to me decent to just rename the unknown class in the test:

```golo
function unknown_class = -> johndoe.class
```

### CODE REVIEW

**Comment by jponge** [comment]
Thanks @Artpej
